### DISCUSSION TITLE
Nuovo valore platform Integration e offset iniziale

### DISCUSSION THREAD

**Discussion by danygi79** [indomus/forum#2542] [HUB: Home Assistant]
Buonasera,
per motivi che non sto qui a spiegare sono costretto a passare da un sensore prelevato da un integrazione (goodwe) al calcolo dello stesso valore con la funzione "Integration - Riemann sum integral" ma vorrei far partire il valore calcolato dal valore del vecchio sensore e non da 0.

La domanda quindi è: c'è un modo per aggingere questo "offset" di avvio?

Grazie a chi eventualmete mi da un aiuto.

**Comment by 50494554524F** [comment]
creati l'altro sensore e poi un sensore somma dei due...
